src/page/forhandsvisning/forhandsvisning.tsx

```diff
@@ -28,10 +28,9 @@ export function Forhandsvisning() {
 	const { utkast, setUtkast, features, setFattedeVedtak } = useDataStore();
 	const { showModal, hideModal } = useModalStore();
 	const { showVarsel } = useVarselStore();
-	const { innsatsgruppe, resetSkjema } = useSkjemaStore();
+	const { innsatsgruppe, resetSkjema, setHarForsoktAForhandsvise } = useSkjemaStore();
```

Comment:
`harForsoktAForhandsvise` og `setHarForsoktAForhandsvise` brukes for sjekke om man skal vise feilmeldinger i skjemaet på om bruker har fylt ut felt eller ikke. Grunnen til at det er en egen variabel for det er fordi vi skal kun vise feilmeldingene når man trykker på forhåndsvisknappen, dvs at man har forsøkt å forhåndsvise.

Reply:
En alternativ måte å gjøre dette på er å bruke `showSection` men endre på typen slik at istedenfor kun true/false så har man et tredje alternativ som f.eks `null`. Hvis man sjekker på utkast siden om `showSection` er null, så kan man initialisere den med en verdi basert på innsatsgruppen som er satt i skjemaet. Siden man sjekker på om verdien er `null` før man gjør endringer, så vil ikke `showSection` overskrives når man går til forhåndsvisning og tilbake.